client/components/ui/navigation/sidebar/Sidebar.vue

```diff
@@ -0,0 +1,159 @@
+<script lang="ts" setup>
+const { t } = useI18n()
+const color = useColorMode()
+const { user } = storeToRefs(useAuth())
+
+const sidebar = toReactive(useLocalStorage('sidebar', { open: true }))
+const sidebarRef = ref<HTMLElement | null>(null)
+
+const isSidebarHovered = useElementHover(sidebarRef)
+const isAdmin = computed(() => user.value?.role === 'ADMIN' || user.value?.role === 'OWNER')
+const { isDebugEnabled } = useDebug()
+
+const menuIcon = computed(() => {
+  if (sidebar.open) {
+    return 'i-tabler-indent-decrease'
+  } else if (isSidebarHovered.value) {
+    return 'i-tabler-menu-2'
+  } else {
+    return 'i-hedera-logo'
+  }
+})
```

Comment:
unused (should be used instead of the v-ifs ?)

Reply:
No, I just forgot to remove this.